.github/workflows/ci.yml

```diff
@@ -43,11 +43,31 @@ jobs:
       - name: Checkout Repository
         uses: actions/checkout@v2
 
-      - name: Set Up Python ${{ matrix.python-version }}
+      - name: Setup Python ${{ matrix.python-version }}
         uses: actions/setup-python@v2
         with:
           python-version: ${{ matrix.python-version }}
 
+      - name: Setup Go
+        uses: actions/setup-go@v3
+        with:
+          go-version: 1.x
+      
+      - name: Install mkcert on macOS
+        if: runner.os == 'macOS'
+        run: |
+          go install filippo.io/mkcert@latest
```

Comment:
Can we use a specific release of mkcert so that this doesn't contribute to indeterministic behavior in truststore's tests?